autotelic/fastify-opentelemetry
# chore: Update example app

### Summary 

 - Update example app to `await fastify.register(...)`
 - Update README and example app's OpenTelemtry SDKs to use latest packages
   - `@opentelemetry/tracing` -> `@opentelemetry/sdk-trace-base`
   - `@opentelemetry/node` -> `@opentelemetry/sdk-trace-node`
   - `TraceIdRatioBasedSampler` now exported from `@opentelemetry/sdk-trace-base`
 
### Test plan

 - Run the example app locally
 - Navigate to `http://localhost:3000` in the browser
 - Confirm spans are logged as expected
